backends-clickhouse/src/main/scala/org/apache/gluten/backendsapi/clickhouse/CHBackend.scala

```diff
@@ -175,7 +176,7 @@ object CHBackendSettings extends BackendSettingsApi with Logging {
       fields: Array[StructField],
       rootPaths: Seq[String],
       properties: Map[String, String],
-      serializableHadoopConf: Option[SerializableConfiguration] = None): ValidationResult = {
+      hadoopConf: Configuration): ValidationResult = {
```

Comment:
The serialize code is added for viewfs support as we need to pass some info to Velox. Not sure if the new code can be ser/de-ser and passed down. 
The code itself is correct, but there is no viewfs related tests in GHA now so we may need to check this manually

Cc @wangyum @JkSelf

Reply:
But from the code logic, I did not find that velox uses the hadoop conf here.

Reply:
@wangyum Could you please help in verifying this PR in your ViewFS environment? My ViewFS test setup is currently not functioning properly.

Reply:
OK. will test it soon.

Reply:
I have tested this patch on our cluster and it still works.